Sources/SwiftASN1/ASN1.swift

```diff
@@ -1254,12 +1254,17 @@ extension FixedWidthInteger {
         }
 
         self = 0
-        let shiftSizes = stride(from: 0, to: bytes.count * 8, by: 8).reversed()
+
+        // Unchecked subtraction because bytes.count must be positive, so we can safely subtract 8 after the
+        // multiply. The same logic applies to the math in the loop. Finally, the multiply can be unchecked because
+        // we know that bytes.count is less than or equal to bitWidth / 8, so multiplying by 8 cannot possibly overflow.
```

Comment:
What about the edge case where `bytes.count == 0`? This would produce `-8` right? We wouldn't do any shifting below which looks good, just wanted to double check

Reply:
Yeah, I'm counting 0 as positive, 0 - 8 is safe for all integers.